Samples/StyleTransfer/AppViewModel.cs

```diff
@@ -0,0 +1,546 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Windows.Media.Core;
+using GalaSoft.MvvmLight.Command;
+using Windows.Media.Capture.Frames;
+using System.Diagnostics;
+using Windows.Media.Capture;
+using Windows.Media.Playback;
+using Windows.Media.Effects;
+using Windows.Media;
+using Windows.Foundation.Collections;
+using Microsoft.AI.MachineLearning;
+using Windows.Storage;
+using System.Runtime.CompilerServices;
+using Windows.Graphics.Imaging;
+using Windows.UI.Xaml.Media.Imaging;
+using Windows.UI.Xaml.Media;
+using Windows.Media.MediaProperties;
+using Windows.Storage.Pickers;
+using Windows.UI.Xaml.Controls;
+using System.IO;
+using Windows.System.Display;
+using Windows.UI.Core;
+using GalaSoft.MvvmLight.Threading;
+using System.Threading;
+using StyleTransferEffectCpp;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Devices.Enumeration;
+
+namespace StyleTransfer
+{
+    class AppViewModel : INotifyPropertyChanged
+    {
+        public AppViewModel()
+        {
+            _appModel = new AppModel();
+            SaveCommand = new RelayCommand(SaveOutput);
+            SetMediaSourceCommand = new RelayCommand<string>(async (x) => await SetMediaSource(x));
+            SetModelSourceCommand = new RelayCommand(async () => await SetModelSource());
+            ChangeLiveStreamCommand = new RelayCommand(async () => await ChangeLiveStream());
+
+            _inputSoftwareBitmapSource = new SoftwareBitmapSource();
+            _outputSoftwareBitmapSource = new SoftwareBitmapSource();
+            _saveEnabled = true;
+            NotifyUser(true);

```

Comment:
>NotifyUser [](start = 12, length = 10)

isnt this a little early, shouldnt it be at the end of initialization?

Reply:
move to end of initalization